weblate/machinery/google.py

```diff
@@ -99,3 +101,46 @@ def get_error_message(self, exc):
                 pass
 
         return super().get_error_message(exc)
+
+
+class GoogleTranslationAdvanced(MachineTranslation):
```

Comment:
I'd probably call it `GoogleTranslationV3`.

Reply:
I didn't know if number in class name does not break any good practice. Yes, i will go with this suggestion.

Reply:
There is nothing "Advanced" on the new API, it's just evolution of it. There will come v4 and v5, and the naming should be prepared for that.

There is no problem in using numbers in class names, it just can't be first letter.

Reply:
It is not in any way advanced for us, i agree. They themselves call it in docs advanced, thus i attempted. I understand now, i will update as suggested :)